pkg/metrics/service-monitor.go

```diff
@@ -96,7 +112,7 @@ func GenerateServiceMonitor(s *v1.Service) *monitoringv1.ServiceMonitor {
 func populateEndpointsFromServicePorts(s *v1.Service) []monitoringv1.Endpoint {
 	var endpoints []monitoringv1.Endpoint
 	for _, port := range s.Spec.Ports {
-		endpoints = append(endpoints, monitoringv1.Endpoint{Port: port.Name})
+		endpoints = append(endpoints, monitoringv1.Endpoint{Port: port.Name, Path: "/metrics"})
```

Comment:
Is it really needed to add the path here? The Prometheus will by default try to fetch the data from the `/metrics` path, and it does work fine without it on my system for custom metrics.

Reply:
Hmm, I might be wrong about this one then :thinking: Because of the other issues I hit with labelling, it's possible that I erroneously thought this also had to be fixed.

Reply:
:+1: 

No worries. I'd drop it from the PR and check if it still works for you. If not, I guess we'll look into that. One would think this would be discovered as a problem long time ago if it didn't work properly... :man_shrugging: